Neuro/NeuroPlugin.cs

```diff
@@ -1,208 +1,48 @@
-﻿using BepInEx;
-using BepInEx.Configuration;
+﻿using System.Reflection;
+using BepInEx;
 using BepInEx.Unity.IL2CPP;
 using HarmonyLib;
+using Neuro.Arrow;
+using Neuro.DependencyInjection;
+using Neuro.Minigames;
+using Neuro.Movement;
+using Neuro.Pathfinding;
+using Neuro.Recording;
+using Neuro.Tasks;
+using Neuro.Utilities;
+using Neuro.Vision;
 using Reactor;
-using UnityEngine;
-using System.Collections.Generic;
-using System.Collections;
-using System.Text.Json;
+using Reactor.Utilities;
 
 namespace Neuro;
 
 [BepInAutoPlugin]
 [BepInProcess("Among Us.exe")]
 [BepInDependency(ReactorPlugin.Id)]
-public partial class NeuroPlugin : BasePlugin
+public partial class NeuroPlugin : BasePlugin, IContextProvider
 {
-    public Harmony Harmony { get; } = new(Id);
+    public static NeuroPlugin Instance => NeuroPlugin.Instance;
```

Comment:
Fixes crash from recursive property.
```suggestion
    public static NeuroPlugin Instance;
```

Reply:
Lmao oops